docs/tutorials/slack-history.md

```diff
@@ -0,0 +1,90 @@
+# Saving Slack Messages
+
+## Overview
+
+The [slack free-tier](https://slack.com/pricing/paid-vs-free) saves only the last 10K messages. For social Slack instances, it may be impractical to upgrade to a paid plan to retain these message. Similarly, for an OSS project like Airbyte where we interact with our community through a public slack instance, the cost of paying for a seat for every slack member is inhibitive.
```

Comment:
```suggestion
The [slack free-tier](https://slack.com/pricing/paid-vs-free) saves only the last 10K messages. For social Slack instances, it may be impractical to upgrade to a paid plan to retain these messages. Similarly, for an OSS project like Airbyte where we interact with our community through a public slack instance, the cost of paying for a seat for every slack member is inhibitive.
```